Generator/Sources/NeedleFramework/Generating/Pluginized/PluginExtensionSerializerTask.swift

```diff
@@ -36,7 +36,7 @@ class PluginExtensionSerializerTask : AbstractTask<SerializedProvider> {
     override func execute() -> SerializedProvider {
         let content = PluginExtensionContentSerializer(component: component).serialize()
         let registration = PluginExtensionRegistrationSerializer(component: component).serialize()
-        return SerializedProvider(content: content, registration: registration)
+        return SerializedProvider(content: content, registration: registration, attributes: [:])
```

Comment:
nit: seems like a lot of these take empty attributes, you could add a default argument so this argument can be elided.